app/gosqs/send_message_test.go

```diff
@@ -0,0 +1,111 @@
+package gosqs
+
+import (
+	"net/http"
+	"net/url"
+	"testing"
+
+	"github.com/Admiral-Piett/goaws/app"
+	"github.com/Admiral-Piett/goaws/app/models"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSendMessage_MaximumMessageSize_Success(t *testing.T) {
+	req, err := http.NewRequest("POST", "/", nil)
+	if err != nil {
+		t.Fatal(err)
```

Comment:
`assert.Nil`

Reply:
OK!
To be honest, I don't usually write Go at work, so I think there are some aspects of the culture that I don't understand.
For testing, `assert.Nil` is often used rather than `t.Fatal`, right?

Reply:
No worries!  I appreciate your flexibility here, especially on my nitpicks! 😄   

I like `assert.<whatever>` since that's a benefit of the `testify` library.  It makes the error states more uniform and easier to decipher, for me anyway.  `t.Fatal` / `t.Error` is Go's native testing pattern, but I think it's more clunky to deal with and lot trickier to figure out which statement is the issue if there are several in a single test.  It also requires a lot of hardcoded error codes, etc., which I don't think are valuable.